cmd/vl3-nse/service_registry.go

```diff
@@ -108,7 +108,7 @@ func (s *serviceRegistry) RegisterWorkload(ctx context.Context, workloadLabels m
 	logrus.Infof("Sending workload register request: %v", serviceWorkload)
 	_, err = s.registryClient.RegisterWorkload(ctx, serviceWorkload)
 	if err != nil {
-		logrus.Errorf("service registration not successful: %w", err)
+		logrus.Errorf("service registration not successful: %v", err)
```

Comment:
When I tried running the unit tests I wrote for aws_tgw_connect.go there was a build failure for this file that was preventing my tests due to having the %w instead of %v.

Reply:
Most likely a forgotten formatting directive that was probably copy-pasted from other `fmt.Errorf` call. The `%w` is a special specified for errors: https://golang.org/pkg/fmt/#Errorf